services/api/routes/public.js

```diff
@@ -40,6 +42,178 @@ var routes = [
       }
     }
   }, {
+    path: '/api/projects',
+    method: 'get',
+    handler: projects.get.all,
+    config: {
+      validate: {
+        query: {
+          count: Joi.number().min(1).max(100).default(10),
+          page:Joi.number().min(1).default(1)
+        }
+      },
+      pre: [
+        prerequisites.calculateOffset
+      ],
+      cors: {
+        methods: ['get', 'options']
+      }
+    }
+  }, {
+    path: '/api/users/{user}/projects',
```

Comment:
just a way to namespace routes, since we can have multiple "services" running on the server.

I never liked em, so I can remove them